### PR TITLE
feature: add tracking event to loggly

### PR DESCRIPTION
**Pull request by hoathaivan**
image

### CODE REVIEW

File: cnext_app/package.json [hunk 1]
```diff
@@ -52,7 +52,6 @@
         "inline-style-prefixer": "^4.0.0",
         "jquery-ui": "^1.13.1",
         "moment-timezone": "^0.5.34",
-        "nanoid": "^3.1.23",
```
Comment (mbui0327) [inline cnext_app/package.json:55]: we need to add this back. if you sync with main, it should have this.

File: cnext_server/__main__.py [hunk 2]
```diff
@@ -164,22 +166,43 @@ def run_help(choice):
         - cnext                        : RESUME APPLICATION or START
         - cnext 8888                   : RESUME APPLICATION at PORT 8888
         - cnext -v                     : Show the Version
+        - cnext -l on/off              : Enable/Disable Loggly Event
```
Comment (mbui0327) [inline cnext_server/__main__.py:169]: chang the text to: `enable/disable event log`

File: cnext_server/__main__.py [hunk 3]
```diff
@@ -164,22 +166,43 @@ def run_help(choice):
         - cnext                        : RESUME APPLICATION or START
         - cnext 8888                   : RESUME APPLICATION at PORT 8888
         - cnext -v                     : Show the Version
```
Comment (mbui0327) [inline cnext_server/__main__.py:168]: change this to `show the version`

File: cnext_server/server/EventLog.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            function (error, response, body) {
+                if (!error && response.statusCode == 200) {
+                    // console.log("send to loggly", body);
+                }
+            }
+        );
+    }, timeout);
+}
+
+module.exports = { logglyEvent };
```
Comment (mbui0327) [inline cnext_server/server/EventLog.js:37]: change this to `eventLog`

File: cnext_server/server/settings.json [hunk 5]
```diff
@@ -0,0 +1 @@
+{"loggly-event": true}
```
Comment (mbui0327) [inline cnext_server/server/settings.json:1]: do we need this?

Comment (hoathaivan) [reply cnext_server/server/settings.json:1]: yes. we need this to store the log status when the user run the 2nd time

File: cnext_server/server/EventLog.js (hunk 4 above)
Comment (hoathaivan) [reply cnext_server/server/EventLog.js:37]: changed

File: cnext_server/__main__.py (hunk 2 above)
Comment (hoathaivan) [reply cnext_server/__main__.py:169]: changed

**Pull request closed without merging**